src/components/conversationCreate/conversationCreate.styles.scss

```
@@ -0,0 +1,702 @@
+		background: var(--m3-error-container, #ffdad6);
+		border-radius: 12px;
+		color: var(--m3-on-error-container, #410002);
+		display: flex;
+		gap: 12px;
+		justify-content: space-between;
+		margin: 0;
+		padding: 12px 16px;
+
+		button {
+			background: none;
+			border: 1px solid currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)